salt/modules/win_system.py

```diff
@@ -817,12 +848,15 @@ def set_system_time(newtime):
     :rtype: bool
     '''
     # Get date/time object from newtime
-    dt_obj = salt.utils.date_cast(newtime)
```

Comment:
Yes, I installed timelib, but date_cast still does not support all of the formats set_system_time and set_system_date are documented to support. Some formats work correctly but not all formats do, and some formats are interpreted incorrectly. For example, set_system_date is documented to support MM-DD-YYYY, but date_cast will try to parse input in that format to DD-MM-YYYY.